back/app/models/custom_field.rb

```diff
@@ -28,6 +28,7 @@
 #  minimum_select_count   :integer
 #  random_option_ordering :boolean          default(FALSE), not null
 #  page_layout            :string
+#  dropdown_layout        :boolean          default(FALSE), not null
```

Comment:
I'm not sure that this attribute _should_ be set by default, as I'm not sure what it means for, say, a page to have either a `true` or `false` value for `dropdown_layout`.

On the other hand, I see we have done this for other attributes that similarly make little sense outside a specific context, e.g. `random_option_ordering`.

Your approach is simpler, as it avoids a data migration to set some value for all existing select & multiselect custom_fields, validation of presence of a boolean for only select and multiselect fields, and the tests to go along with the validation.

So, on balance, I can't really think why this would cause any real-world issues, and thus I don't feel strongly about it.